### PR TITLE
zookeeper: add documentation

### PR DESCRIPTION
**Pull request by rishabh96b**
This PR adds the documentation of KUDO zookeeper. The docs of following topics have been added-
1. Installation
2. Configuration
3. Limitations

### CODE REVIEW

File: repository/zookeeper/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Zookeeper Operator
+
+The KUDO Zookeeper operator creates, configures and manages [Apache Zookeeper](https://zookeeper.apache.org/) clusters running on Kubernetes.
+
+## Getting started
+
+The latest stable version of Zookeeper operator is `0.2.0`
+
+For the latest master branch you can check  [docs](./docs/latest) docs
```
Comment (zmalik) [inline repository/zookeeper/README.md]: we removed this line from Kafka docs also, so let's remove from here also and only give the latest stable version docs

File: repository/zookeeper/docs/latest/configuration.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+To deploy Kafka using a different storage class, you can use the parameter `STORAGE_CLASS`
+
+```
+kubectl kudo install zookeeper --instance=my-zookeeper-name -p STORAGE_CLASS=<STORAGE_CLASS_NAME>
+```
+
+##### Docker image
+
+The Dockerfile used to build the KUDO Zookeeper operator is hosted at [Zookeeper's Dockerhub Repository](https://hub.docker.com/_/zookeeper).
```
Comment (zmalik) [inline repository/zookeeper/docs/latest/configuration.md]: let's add a link to `Dockerfile` also

**Review by zmalik — CHANGES_REQUESTED** [review]
thanks @rishabh96b ! left few comments!

File: repository/zookeeper/docs/latest/configuration.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# Configuration 
+
+### Resources
+
+By default, KUDO Zookeeper resource configuration is set to the minimum recommended values for production usage. 
+But users can and should tune the configurations based on the workload requirements of their Kafka cluster.
```
Comment (samvantran) [inline repository/zookeeper/docs/latest/configuration.md]:
```suggestion
But users can and should tune the configurations based on the workload requirements of whatever service makes use of Zookeeper. 
```

File: repository/zookeeper/docs/latest/configuration.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+            -p CPUS=1 \
+            -p NODE_COUNT=5 \
+            -p MEMORY=1Gi \
+            -p DISK_SIZE=5Gi \
+```
+
+##### Ports
+
+The parameter `CLIENT_PORT`(default: 2181) sets the port for listening to client requests.
+Similarly, `SERVER_PORT`(default: 2888) is used to set port on which zookeeper will listen for requests from other servers in the ensemble and `ELECTION_PORT`(default: 3888) can be used to set the port on which the Zookeeper process will perform leader election.
```
Comment (samvantran) [inline repository/zookeeper/docs/latest/configuration.md]:
```suggestion
Similarly, `SERVER_PORT`(default: 2888) is used to set the port which zookeeper will listen on for requests from other servers in the ensemble and `ELECTION_PORT`(default: 3888) can be used to set the port on which the Zookeeper process will perform leader election.
```

File: repository/zookeeper/docs/latest/configuration.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+
+##### Storage
+
+By default, the Zookeeper operator will use the default storage class of the Kubernetes cluster. 
+
+To deploy Kafka using a different storage class, you can use the parameter `STORAGE_CLASS`
```
Comment (samvantran) [inline repository/zookeeper/docs/latest/configuration.md]:
```suggestion
To deploy Zookeeper using a different storage class, you can use the parameter `STORAGE_CLASS`
```

**Review by samvantran — APPROVED** [review]
Looks good @rishabh96b ! Just a few comments from me but approving ahead of time because the rest is 👌

File: repository/zookeeper/docs/latest/configuration.md (hunk 3 above)
Comment (rishabh96b) [reply repository/zookeeper/docs/latest/configuration.md]: Awesome. This is more generic information.

**Review by shubhanilBag — APPROVED** [review]

**Review by zmalik — APPROVED** [review]

**Pull request merged**